src/main/java/Bellroy/Storage.java

```diff
@@ -0,0 +1,83 @@
+package Bellroy;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.FileReader;
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<Task> loadTasks() throws IOException {
+        List<Task> taskList = new ArrayList<>();
+        File file = new File(filePath);
+        if (!file.exists()) {
+            file.createNewFile();
+        } else {
+            BufferedReader reader = new BufferedReader(new FileReader(filePath));
+            String line = reader.readLine();
+            while (line != null) {
+                char taskType = line.charAt(1);
+                boolean isDone = line.charAt(4) == 'X';
+                String remainder = line.substring(7).trim();
+
+                switch (taskType) {
+                    case ('T'):
+                        Task tempToDo = new Todo(remainder);
+                        if (isDone) {
+                            tempToDo.markDone();
+                        }
+                        taskList.add(tempToDo);
+                        break;
+                    case ('D'):
+                        String[] deadlineParts = remainder.split(" \\(by: ");
```

Comment:
Perhaps you can also add some inline comments on what this chunk of code snippet does as it would look overwhelming at first glance for readers :)